src/api/libcellml/units.h

```diff
@@ -367,9 +376,19 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     size_t unitCount() const;
 
-private:
-    void swap(Units &rhs); /**< Swap method required for C++ 11 move semantics. */
+    /**
+     * @brief Return the scaling factor difference between two units.
+     * 
+     * This can be interpreted as factor, where u1 = (factor)*u2.  This method
+     * does not check to see if the units are compatible.
+     * 
+     * @param units1 The first units to compare.
+     * @param units2 The second units to compare.
+     * @return The factor units1/unitsf2.
+     */
+    static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
```

Comment:
It might be nice to also have a non-static version of `scalingFactor()`, i.e.

```
double scalingFactor(const UnitsPtr &nits);
```

to return the scaling factor difference between the current and the given units.

Reply:
see #447.